package.json

```diff
@@ -87,6 +87,7 @@
     "lint": "npm run lint:css && npm run lint:js",
     "lint:js": "eslint .",
     "lint:css": "stylelint public/css/",
+    "lint:fluent": "pip3 install -r .github/requirements.txt && moz-fluent-lint ./locales/en --config .github/linter_config.yml",
```

Comment:
Good point. Not only it would try to install it every time, but it would not update it if already installed.

One possible alternative is to use a virtualenv, and call a bash script here ([example](https://github.com/flodolo/mozpm_stats/blob/main/firefox_stats/extract_stats.sh)). Then just put python and virtualenv as requirements in the README?

Reply:
Good point, guys. Separated it into readme + command